companion/src/mainwindow.cpp

```diff
@@ -1019,18 +1019,11 @@ void MainWindow::sdsync()
 }
 
 void MainWindow::changelog()
-{
-  ReleaseNotesDialog * dialog = new ReleaseNotesDialog(this);
```

Comment:
This class is no longer used so maybe remove the files also?  

Though of course this is a better name than "ReleaseNotesFirmwareDialog" since that one now shows both.  :)